services/api/src/resources/project/sql.ts

```diff
@@ -29,6 +29,26 @@ export const Sql = {
     knex('project as p')
       .whereIn('p.id', projectIds)
       .toString(),
+  deleteEnvironmentVariables: (id: number) =>
+    knex('env_vars')
+      .where('project', '=', id)
+      .delete()
+      .toString(),
+  deleteDeployTargeTConfigs: (id: number) =>
```

Comment:
nitpick: weird capitalization `deleteDeployTargeTConfigs` > `deleteDeployTargetConfigs`. Also needs fixed in services/api/src/resources/project/helpers.ts